apps/admin-portal/src/components/identity-providers/meta/templates.ts

```diff
@@ -0,0 +1,56 @@
+import { IdentityProviderTemplateListItemInterface } from "../../../models";
```

Comment:
Shall we add the import after the copyright header.

Reply:
removed